aea/configurations/base.py

```diff
@@ -367,6 +568,19 @@ def __init__(
         self.description = description
         self.config = config
 
+    @property
+    def component_type(self) -> ComponentType:
+        """Get the component type."""
+        return ComponentType.CONNECTION
+
+    @property
+    def package_dependencies(self) -> Set[ComponentId]:
+        """Get the connection dependencies."""
+        return set(
+            ComponentId(ComponentType.PROTOCOL, protocol_id)
+            for protocol_id in self.protocols
```

Comment:
in our model, connections can only have protocols as dependencies. In the future, they might depend on some "skills"?
Think about packages that only contain utilities. Instead of deploying them to PyPI, they can be deployed to the registry and used by other agents as local packages.

Reply:
Open issue for this?

Reply:
https://github.com/fetchai/agents-aea/issues/960